addons/l10n_in/models/account_tax.py

```diff
@@ -61,13 +61,12 @@ def _l10n_in_get_hsn_summary_table(self, base_lines, display_uom):
                 rounding_method='round_per_line',
                 product=product,
             )
-
             # Rate.
-            rate = sum(
-                tax_data['tax'].amount
-                for tax_data in taxes_computation['taxes_data']
-                if tax_data['tax']['l10n_in_tax_type'] in ('igst', 'cgst', 'sgst')
-            )
+            unique_taxes_data = set()
+            for tax_data in taxes_computation['taxes_data']:
+                if tax_data['tax']['l10n_in_tax_type'] in ('igst', 'cgst', 'sgst'):
+                    unique_taxes_data.add(tax_data['tax'])
```

Comment:
We can just do set(t['tax'] for t in taxes_computation['taxes_data'] if t['tax']['l10n_in_tax_type'] in ...